utils/spdx-document/src/test/kotlin/model/SpdxDocumentTest.kt

```diff
@@ -33,8 +33,8 @@ import org.ossreviewtoolkit.utils.test.readResource
 /**
  * This test uses the following test assets copied from the SPDX 2.2.2 specification examples.
  *
- * 1. https://github.com/spdx/spdx-spec/blob/development/v2.2.2/examples/SPDXYAMLExample-2.2.spdx.yaml
- * 2. https://github.com/spdx/spdx-spec/blob/development/v2.2.2/examples/SPDXJSONExample-v2.2.spdx.json
+ * 1. https://github.com/spdx/spdx-spec/blob/v2.2.2/examples/SPDXYAMLExample-2.2.spdx.yaml
```

Comment:
It seems that the YAML file contents below do not *exactly* match the downloaded file, but have indentation changed. I think it'd be better to take the file exactly as-is from upstream to ease diffing / comparison by hash.

Reply:
Ah right, these came from IntelliJ which applies it automatically when pasting text. I've fixed it.